### PR TITLE
LLVM linters set correct arch for linux arm64

### PR DESCRIPTION
**Pull request by pat-trunk-io**

### CODE REVIEW

File: linters/clang-format/plugin.yaml [hunk 1]
```diff
@@ -8,6 +8,11 @@ lint:
         - os: macos
           url: https://trunk.io/releases/clang-format-${version}-macos-x86_64.tar.gz
           version: <14.0.0
+        - os:
+            linux: linux
+          cpu:
+            arm_64: aarch64
+          url: https://trunk.io/releases/clang-format-${version}-${os}-${cpu}.tar.gz
```
Comment (TylerJang27) [inline linters/clang-format/plugin.yaml:15]: Can you modify this to make this and the following block be "correct" (more than just first-match)?

File: linters/clang-tidy/plugin.yaml [hunk 2]
```diff
@@ -8,6 +8,11 @@ lint:
         - os: macos
           url: https://trunk.io/releases/clang-tidy-${version}-macos-x86_64.tar.gz
           version: <14.0.0
+        - os:
+            linux: linux
+          cpu:
+            arm_64: aarch64
+          url: https://trunk.io/releases/clang-tidy-${version}-${os}-${cpu}.tar.gz
```
Comment (TylerJang27) [inline linters/clang-tidy/plugin.yaml:15]: same as above

File: linters/iwyu/plugin.yaml [hunk 3]
```diff
@@ -4,6 +4,11 @@ lint:
     - name: include-what-you-use
       version: 0.19
       downloads:
+        - os:
+            linux: linux
+          cpu:
+            arm_64: aarch64
+          url: https://trunk.io/releases/include-what-you-use-${version}-${os}-${cpu}.tar.gz
```
Comment (TylerJang27) [inline linters/iwyu/plugin.yaml:11]: same as above

File: linters/clang-format/plugin.yaml (hunk 1 above)
Comment (pat-trunk-io) [reply linters/clang-format/plugin.yaml]: Done

File: linters/clang-tidy/plugin.yaml (hunk 2 above)
Comment (pat-trunk-io) [reply linters/clang-tidy/plugin.yaml]: Done

**Review by TylerJang27 — APPROVED** [review]
LGTM

**Pull request merged**